### DISCUSSION TITLE
Export data to Mysql

### DISCUSSION THREAD

**Discussion by FrankWoody** [DaxStudio/DaxStudio#873] [Q&A]
Hello everyone, I hope you are all well.
I'm developing a project that I need to track the progress of my activities. The connection to my database takes place in real time directly from Ms planner, to do this monitoring so I thought of the following method.
1st I have a table that connects me directly with planner. So this is tb1.
2nd I duplicated the tb1 and created the tb2, in the tb2 I disabled the update so that it would serve as a history that I can use to compare the progress of the tb1.
3rd However, it may be possible to add new tasks on tb1 that I don't have on tb2, so I need to add this information on tb2 and save this information in it for the next updates.
I've thought about using dax functions to create custom tables, but it still didn't work out, I wanted to know if it's possible to do this manipulation with Dax or Powerquery. Share ideas and methods I'm open to suggestions Thank you all.
This was my idea to discover that through daxstudio I can export information directly from Bi to the Sql Server. So I was wondering if I can program dax studio to do an incremental update of the information straight from power bi

**Comment by dgosbell** [comment]
> So I was wondering if I can program dax studio to do an incremental update of the information straight from power bi

The short answer is no, this is not possible to do with DAX Studio.

In order to do an incremental update DAX Studio would need to query both the current dataset as well as the source system and then compare the two to figure out what is different. DAX Studio only connects to tabular datasets so it cannot do this. Normally I would build logic like this in the source system using something like a stored procedure if it was a SQL based source.
